INSTALL.md

```diff
@@ -34,91 +35,41 @@ apt-get update
 
 ### Installation
 ```bash
-wget https://github.com/mysteriumnetwork/node/releases/download/{VERSION}/mysterium-node_linux_amd64.deb
-sudo dpkg --install --force-depends mysterium-node_linux_amd64.deb
+wget https://github.com/mysteriumnetwork/node/releases/download/{VERSION}/myst_linux_amd64.deb
+sudo dpkg --install --force-depends myst_linux_amd64.deb
 sudo apt-get install -y --fix-broken
 ```
+
 ### Running service
 ```bash
 sudo service mysterium-node start
 sudo service mysterium-node status
 ```
+
 ### Debugging service
 ```bash
 sudo tail -f /var/log/mysterium-node/*
 ```
-### Debugging standalone
-```bash
-sudo mysterium_server --data-dir=/var/lib/mysterium-node --config-dir=/etc/mysterium-node --runtime-dir=/tmp --identity=0x123456..
-```
-
-
-## Mysterium VPN client (Debian && Ubuntu) - tested on Ubuntu 16.04
-### Download
- * https://github.com/mysteriumnetwork/node/releases/download/{VERSION}/mysterium-client_linux_amd64.deb
-
-### Add latest OpenVPN repository
-
-```bash
-apt-get update && apt-get install -y curl
-curl -s https://swupdate.openvpn.net/repos/repo-public.gpg | apt-key add && echo "deb http://build.openvpn.net/debian/openvpn/stable xenial main" > /etc/apt/sources.list.d/openvpn-aptrepo.list && rm -rf /var/cache/apt/* /var/lib/apt/lists/*
-apt-get update
-```
-
-### Installation
-```bash
-wget https://github.com/mysteriumnetwork/node/releases/download/{VERSION}/mysterium-client_linux_amd64.deb
-sudo dpkg --install --force-depends mysterium-client_linux_amd64.deb
-sudo apt-get install -y --fix-broken
-```
-
-In order for mysterium client to change system DNS servers to ones provided by VPN node
- it needs to modify /etc/resolv.conf
-This change is performed by resolvconf utility. After resolvconf utility is installed, you need:
-```bash
-dpkg-reconfigure resolvconf
-```
-in order to be able to alter /etc/resolv.conf dinamically.
 
-### Running service
-```bash
-sudo service mysterium-client start
-sudo service mysterium-client status
-```
-### Debugging service
-```bash
-sudo tail -f /var/log/mysterium-client/*
-```
 ### Debugging standalone
 ```bash
-sudo mysterium_client --data-dir=/var/lib/mysterium-client --config-dir=/etc/mysterium-client --runtime-dir=/tmp
+sudo myst --data-dir=/var/lib/mysterium-node --config-dir=/etc/mysterium-node --runtime-dir=/tmp --identity=0x123456..
 ```
 
-
 ## Mysterium VPN node and client standalone binaries (.tar.gz)
```

Comment:
Why to remove this?

Reply:
Removed section for client download and installation. Since there are no more dedicated client binary and installation process, only instruction for installing single binary left.